AdblockPlusSafari/AdblockPlus.m

```diff
@@ -73,53 +82,42 @@ - (void)setActivated:(BOOL)activated
   [_adblockPlusDetails synchronize];
 }
 
-#pragma mark -
-
-- (NSString *)contentBlockerIdentifier
+- (void)setFilterlists:(NSDictionary<NSString *,NSDictionary<NSString *,NSObject *> *> *)filterlists
```

Comment:
Actually, it is used. Compiler will replace assignments self.filterlists = ...; to [self setFilterlists:...]; for you. It is just syntactical sugar. 

Objective-C is quite old and obsolete language. That is why I would prefer to write content blocker in swift.